src/main/java/twistedgate/immersiveposts/APITest.java

```diff
@@ -19,6 +18,8 @@
  * @author TwistedGate
  */
 //@Mod("apitest")
+	//Useless class? Consider deleting it!
+@SuppressWarnings("all")
 public class APITest{
```

Comment:
My bad, I was not saying "useless" as like "bad code" or something, I was just saying that a class never being called is basically useless. Use a @Annotation to let people know there's a reason why it's still there

Reply:
Figured the class name being API*Test would be enough, since its a simple short name that cuts right to the point of it's use. =P
(While also serving as a sort of example mod, as the description already says)